app/forms/add_user_email_form.rb

```diff
@@ -46,7 +42,7 @@ def email_address_record(email)
   attr_reader :success, :email_address
 
   def valid_form?
-    @allow && valid?
+    valid?
   end
```

Comment:
maybe we don't even need a separate `valid_form?` method now?

Reply:
good call, we do not. removed in 6fb1b69